promql/testdata/operators.test

```diff
@@ -35,6 +35,16 @@ eval instant at 50m - - - SUM(http_requests) BY (job)
 eval instant at 50m - - - 1
   -1
 
+eval instant at 50m -2^---1*3
+  -0.75
```

Comment:
Shouldn't thus be -1.5?

Reply:
Yes, it should.

Reply:
This proves that math students are bad at calculating. q.e.d.